src/style.css

```diff
@@ -1,3 +1,77 @@
+html {
+  margin: 0;
+  display: block;
+
+  --main-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
```

Comment:
Don't use tag names for styling (except `html` and `body`). Instead of styling the `ul` element directly, it's better to apply styles using classes to avoid potential conflicts and enhance reusability. For example, you could use `.nav__list` to apply the `list-style: none;` property.